brigade/core/configuration.py

```diff
@@ -24,6 +24,40 @@
         'default': os.path.join(os.path.expanduser("~"), ".ssh", "config"),
         'default_doc': '~/.ssh/config'
     },
+    'logging_dictConfig': {
```

Comment:
While I realise dictConfig just refers to the Python object, I don't really like the camel case.

Reply:
Yeah, we should. But I don't think it's super important so maybe we can create an issue and fix it later with calm.

Reply:
I don't either but I figured it was better to keep the original name. Can change if you have strong opinions ;)

Reply:
Strong opinions about it might be to go a bit to far. :) Might just be more confusing to rename it just for the sake of renaming.